src/commands.cpp

```diff
@@ -3416,6 +3430,60 @@ void CommandGetUserData::procresult()
     }
 }
 
+CommandGetMiscFlags::CommandGetMiscFlags(MegaClient *client)
```

Comment:
This command is unrelated to this PR. Also, FYI, it's missing the new `smsve` flag, which probably is included in the response.

Reply:
yes, I had removed the smsve related code in this one already.   I think we should keep the command in the PR though, as it is one of the ones that needs to be batched independently, which is what this PR is about, and it's not specific to SMS (except that we had never noticed that we needed it until that project).   I think it's just as easy to review in this PR, and we won't call it until the SMS changes are merged so it won't break anything.   I hope that sounds ok - thanks!

Reply:
Alright, let's keep this new command in the PR. But in general terms, we should avoid introducing unrelated features. It makes code's management easier (in terms of debugging, tracking issues, to guess the root of them, etc) ;)